data-prepper-api/src/main/java/com/amazon/dataprepper/model/event/JacksonEvent.java

```diff
@@ -79,6 +89,7 @@ protected JacksonEvent(final Builder builder) {
         }
 
         this.jsonNode = getInitialJsonNode(builder.data);
+        put(TIMESTAMP_KEY, ZonedDateTime.now());
```

Comment:
What is the user experience if the `@timestamp` already existed in the event? It looks like the exsiting `@timestamp` be overwritten.

I think we should update the JacksonEvent docs to indicate a `@timestamp` field will be added to the object and the expected default behavior for an existing value.

Reply:
I'll update the Java docs for `JacksonEvent`.

Reply:
I am not sure if overwriting the field is the correct thing to do here. This has the potential to erase customer data and impact peer forwarding.